pkg/apis/core/v1/conversion.go

```diff
@@ -156,21 +157,17 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 	// Add field conversion funcs.
 	err = scheme.AddFieldLabelConversionFunc("v1", "Pod",
 		func(label, value string) (string, string, error) {
-			switch label {
-			case "metadata.annotations",
-				"metadata.labels",
-				"metadata.name",
+			path, _ := fieldpath.SplitMaybeSubscriptedPath(label)
```

Comment:
no subscripted path changes in field label conversion

Reply:
Done.